README.md

```diff
@@ -41,3 +41,38 @@ advancement of students in science, technology, engineering, and mathematics.
 |Development hardware| No | All | Each member will require a computer capable of running the tools needed to develop the application |
 |Cloud-based hosting environment| No | Dakota (Cody) Kaup | Hosting environment to be used for development and testing|
 |CSS framework | No | Joe | A CSS framework will be used to provide additional UI controls and site theming |
+
+## Project Metholodology
```

Comment:
I would move this entire Methodology section to between the Risk List and the Resources needed to match the milestone 1 rubric. It's a really picky thing, but it's better to be safe than sorry!
https://github.com/MLHale/CYBR4580-8950/blob/master/projects/milestone1.md